addons/ai_apps/static/src/message_actions.js

```diff
@@ -0,0 +1,64 @@
+import { useComponent, useState } from "@odoo/owl";
+
+import { user } from '@web/core/user';
+import { _t } from "@web/core/l10n/translation";
+import { messageActionsRegistry, transformAction } from "@mail/core/common/message_actions";
+
+messageActionsRegistry
+    .add("insertToComposer", {
+        condition: (component) => (
+            component.props.thread.channel_type === "ai_composer" &&  // only show button in ai_composer chat
+            component.message.Model.insertButtonCaller && (  // after a reload both parts of the below conditions are undefined and but we don't want to button to appear
+                component.message.Model.insertButtonCaller === component.env.chatCaller ||  // only show button when the modal that called the AI is mounted
+                component.env.isSmall
+            ) &&
+            component.message.author.userId !== user.userId  // don't show the buttons for the user's messages
+        ),
+        title: () => _t("Use this"),
+        onClick: (component) => {
+            if (component.env.isSmall) {
+                component.props.insertToEditor(component.props.message.body);
+                component.props.thread.closeChatWindow();
+            } else {
+                const fragment = document.createDocumentFragment();
+                const content_root = document.createElement('p');
+                content_root.innerHTML = component.props.message.body.replace(/^<p>(.*?)<\/p>$/, '$1');
+                fragment.appendChild(content_root);
+                component.env.insertToEditor(fragment);
+            }
+        },
+        sequence: 0,
+    })
+    .add("copy-message", {
+        condition: (component) => component.props.thread.channel_type === "ai_composer",
+        icon: "fa fa-copy",
+        title: _t("Copy to Clipboard"),
+        onClick: (component) => component.message.copyMessageText(),
+        sequence: 1,
+    });;
```

Comment:
That message action will likely come base to `mail`, not that useful on desktop but mobile users might want this. Let's keep there for now!